Source/Core/Core/ConfigManager.cpp

```diff
@@ -374,6 +373,10 @@ bool SConfig::SetPathsAndGameMetadata(const BootParameters& boot)
   if (!std::visit(SetGameMetadata(this, &m_region), boot.parameters))
     return false;
 
+  m_local_config_override = boot.config_override;
+  Config::AddLayer(
+      ConfigLoaders::GenerateLocalGameConfigLoader(m_local_config_override, m_revision));
+
```

Comment:
This looks fishy... wouldn't you want to use the game ID in the not-overriden case?

Reply:
You probably also should move the Config::AddLayer() back to where it was before -- at the very least, it needs to be called before the `DolphinAnalytics::Instance().ReportGameStart();` call.

Reply:
Whoops, good catch that should be a call to GetLocalConfig()